helpers.go

```diff
@@ -157,6 +157,41 @@ func getCassandraType(name string) TypeInfo {
 	}
 }
 
+func splitCompositeTypes(name string) []string {
+	if !strings.Contains(name, "<") {
+		return strings.Split(name, ", ")
+	}
+	var parts []string
+	lessCount := 0
+	segment := ""
+	foundMore := false
+	for _, char := range name {
+		if char == ',' && lessCount == 0 {
+			if segment != "" {
+				parts = append(parts, strings.TrimSpace(segment))
+			}
+			segment = ""
+			continue
+		}
+		segment += string(char)
+		if char == '<' {
+			lessCount++
+		} else if char == '>' {
+			lessCount--
+			foundMore = true
+		}
+		if lessCount == 0 && foundMore {
```

Comment:
I dont see the need for this and `foundMore`. The rest of the logic should handle this case.

Reply:
`foundMore` is necessary because `lessCount` will be zero when maybe an `<` hasn't been found and it shouldn't get in at that point

Reply:
If less count is zero, it will hit a , before a <

Reply:
Not in the case of a regular type for example `tuple<frozen<tuple<int, int>>, int, frozen<tuple<int, int>>>`, after reading `frozen<tuple<int, int>>` when it gets to `int`, `lessCount` will be zero but it shouldn't get in there. Also the tests fail if we remove `foundMore`.